src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs/breadcrumb.rs

```diff
@@ -0,0 +1,416 @@
+//! This module provides a clickable view for a single breadcrumb.
+
+use crate::prelude::*;
+
+use super::GLYPH_WIDTH;
+use super::HORIZONTAL_MARGIN;
+use super::VERTICAL_MARGIN;
```

Comment:
Icons and separators are not displayed in a pixel-perfect way (which was one of the requirements of this task). There is some non-integer translation of pixels somewhere. Please be sure to draw shapes using sizes snapped to full pixels, like width of `8.0` , not `7.6`. Also, do not translate breadcrumbs on X-axis fraction of pixels. If necessary, round the width value of a caption to full pixel values (I marked the fragments that show the improper aliasing on zoom): <img width="1283" alt="aliasing" src="https://user-images.githubusercontent.com/1623053/89509790-08430200-d7d0-11ea-878a-fa6f6dd2aa43.png">

Reply:
@wdanilo I removed the fractional translation and I am not using any fraction elsewhere. It still persists. It might probably be an issue with the SDF as I pointed out earlier about the need to add a padding to the shape size.